fuzzers/aflcc/fuzzer.py

```diff
@@ -249,7 +249,7 @@ def post_build(fuzz_target):
     os.environ['AFL_COVERAGE_TYPE'] = 'ORIGINAL'
     os.environ['AFL_CONVERT_COMPARISON_TYPE'] = 'NONE'
     os.environ['AFL_DICT_TYPE'] = 'NORMAL'
-    bin1_cmd = '{compiler} {flags} -O3 {target}.bc -o ' \
+    bin1_cmd = '{compiler} {flags} -O2 {target}.bc -o ' \
```

Comment:
Is this required ? We build all fuzzers with -O3.

Reply:
ah... I can change back.